pkg/controller/perconaservermongodb/smart.go

```diff
@@ -3,11 +3,8 @@ package perconaservermongodb
 import (
 	"context"
 	"fmt"
-	"sort"
-	"strings"
```

Comment:
strange work of goimports

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/719/commits/de7cb57eba854c2446adfdc36261fbb71ebc1b04